Tools/wasm/README.md

```diff
@@ -281,6 +286,50 @@ make -j$(nproc)
 popd
 ```
 
+## Running Python in WASI
+
+Here is a simple script that can run a python script, it expects that the Python installation
+is at the root of the filesystem (and it appears that currently that is [required](https://github.com/python/cpython/issues/100307).
```

Comment:
Issue https://github.com/python/cpython/issues/100307 is not about _where_ you run the WASI binary from, but _how_ you have to mount the `lib` directory that contains the stdlib.

```suggestion
Here is a script for running a Python file. It helps demonstrate how WASI
follows a capabilities security model, requiring that you explicitly clear which
directories are accessible from within the WASI runtime. You can specify the
_host directories_ that you want to allow in and the _guest directories_ that
they map to.

Make sure to update the `PYTHON_WASI_ROOT` variable in the script to the
directory where you are storing your WASI binary and `lib` directories with the
stdlib.
```